helm/libero-reviewer/templates/deployment-continuum-auth.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "libero-reviewer.fullname" . }}-continuum-auth
+  labels:
+    app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+    helm.sh/chart: {{ include "libero-reviewer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: client
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: client
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: client
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-continuum-auth
+          image: "{{ .Values.images.continuumAuth.repository }}:{{ .Values.images.tag }}"
```

Comment:
this `tag` needs to be `.Values.image.continuum_auth.tag` different now that the Docker images come from different repositories and their commits